app/views/local_authorities/index.html.erb

```diff
@@ -26,10 +26,13 @@
     <% @authorities.each do |authority| %>
       <tr>
         <td>
-          <%= link_to authority.name, local_authority_path(authority.slug, filter: 'broken_links'), class: 'js-open-on-submit name' %>
+          <%= link_to authority.name, local_authority_path(authority.slug, filter: 'broken_links'), class: "js-open-on-submit name authority-status-#{authority.active? ? 'active' : 'inactive'}" %>
           <div class="center count <%= singular_or_plural(authority.broken_link_count) %>">
```

Comment:
Very small suggestion but since we don't need a selector for `authority-status-active` - I wonder if the above should only add the 'inactive' CSS?

`"js-open-on-submit name #{"authority-status-inactive" if !authority.active?}"`